kolibri/core/assets/src/views/StorageNotification.vue

```diff
@@ -0,0 +1,195 @@
+<template>
+
+  <div v-if="bannerOpened" class="banner" :style="{ background: $themeTokens.surface }">
+    <div class="banner-inner">
+      <h1 style="display: none">
+        Storage notification
```

Comment:
This will need to be a string - even though it is not visible and used for accessibility purposes, we will still want it to be translated. Then, the screen reader can present the header information to the user in the language they are using Kolibri in